README.md

```diff
@@ -146,9 +173,13 @@ Then, visit the site at http://localhost:4001/
 
 [Algolia](https://www.algolia.com) powers our search features. Sometimes after content updates or GTFS releases we will find that the search results do not contain up-to-date results. When this happens you can re-index the Algolia data by running: `mix algolia.update`.
 
-## Additional Documentation
+## Additional Resources
 
-See [docs](docs) for information about [testing](docs/TESTING.md) and other development details. There is also documentation for some [features that have come and gone](docs/SeasonalFeatures.md).
+New to the team, or looking for further developer resources?
+- [Repo docs](docs): info about [testing](docs/TESTING.md) and other development details.
+- [Intro to the V3 API](https://github.com/mbta/wiki/blob/master/api/intro.md): a starter guide to the data model with links to relevant API resources.
+- [Dotcom Engineering Guide](https://docs.google.com/document/d/1Vg-8-APtBk7JYuj0TgWvcrWjU5mEA9vCk58aaJgq02Q/edit): a more comprehensive look at the Dotcom ecosystem, the V3 API Data Model, and other project info.
```

Comment:
This Dotcom Engineering Guide is either _internal_ or at least not accessible without an mbtace account...so we either "open up" the document, or we remove a reference to it in this README.md...

Reply:
Oh, I see.  The "Intro to V3 API" doc on our wiki links to it, also.  Is the wiki supposed to be open to the public?

Reply:
https://github.com/mbta/wiki is a private repo so I'm assuming that only CTD has access to it, in which case I think we can leave that reference there.

Reply:
Let's leave both private references (the wiki and google drive links) in here.